eZ/Bundle/EzPublishCoreBundle/EventListener/CacheViewResponseListener.php

```diff
@@ -31,17 +31,25 @@ class CacheViewResponseListener implements EventSubscriberInterface
     private $enableTtlCache;
 
     /**
-     * Default ttl for ttl cache.
+     * Default ttl for ttl cache for OK (200) and OK but empty requests (204, 304).
      *
      * @var int
      */
     private $defaultTtl;
 
-    public function __construct($enableViewCache, $enableTtlCache, $defaultTtl)
+    /**
+     * Default ttl for any requests not considered Ok, but been marked to be cached anyway (but for shorter time).
```

Comment:
~~been~~

"but for shorter time" is not necessarily true, is it? But recommended. Could say "we recommend to set it to a shorter time".